tools/corba/src/main/java/org/apache/cxf/tools/corba/processors/wsdl/WSDLToCorbaHelper.java

```diff
@@ -1431,7 +1431,7 @@ private CorbaType processOMGUnion(XmlSchemaComplexType complex, QName defaultNam
                 } else if (fields.size() == 1) {
                     caselist.add("TRUE");
                 } else {
-                    String msg = "Discriminator Type doesnt match number of Choices in Union:" + name;
+                    String msg = "Discriminator Type doesn't match number of Choices in Union:" + name;
```

Comment:
```suggestion
                    String msg = "Discriminator Type does not match number of Choices in Union:" + name;
```

For log messages, it might be easier to translate "does not" than "doesn't".